src/components/routes/Host/HostListingCard/HostListingCard.tsx

```diff
@@ -16,6 +16,8 @@ interface Props extends HostListingShort {
   deactivateListing: (id: string) => Promise<Listing>;
 }
 
+const INCOMPLETE_LISTING = "This listing is incomplete. Use the Edit button to complete this listing to publish.";
```

Comment:
Maybe add something to the effect of all fields must be filled to publishable?

Reply:
Sure, worked in the phrase "all required fields" (there are a couple of optional fields, so "all fields" seemed misleading)

Reply:
👍